xof/k12/k12.go

```diff
@@ -8,6 +8,7 @@ package k12
 
 import (
 	"encoding/binary"
+	"sync"
```

Comment:
we must also add a CI flow to test with the `-race ` flag enabled.
Note this type of test takes much more time to run.

Reply:
I've added one just for this package.